ui/lib/pki/addon/templates/certificates/index.hbs

```diff
@@ -0,0 +1,75 @@
+<Toolbar>
+  {{#if (gt this.model.length 0)}}
+    <ToolbarFilters>
+      <NavigateInput
+        @filterFocusDidChange={{action "setFilterFocus"}}
```

Comment:
Let me try. This was a code copy from the NavigateInput component (which is old and doesn't have documentation).

Reply:
Okay, well that was interesting. The component wasn't working for this list filter view. I'm going to remove filtering for now as the whole NavigateInput component needs to be glimmerized and refactored so it can be used in an engine (I made a ticket for this and left some comments as to why). Thank you for the comment, I wouldn't have caught this otherwise.